### ISSUE TITLE
Implement Semantic Segmentation using DeepVL3+ on Pascal VOC5i

### ISSUE THREAD

**Issue by Temiloluwa** [Temiloluwa/leo_segmentation#8]
- [x] Generate Pascal VOC 5i dataset 

- [ ] Try different pre-trained models for the decoder
- [ ] We can get outputs from each upsampled layer in the decoder and generate many stages of embeddings
- [ ] -We can also generate embeddings after performing a little transfer learning. 

### References
- https://www.tensorflow.org/tutorials/images/segmentation
- https://github.com/tensorflow/models/tree/master/research/deeplab

### Note
- #4 Correlate the shape of the embeddings with data processing especially the masks
